moc-health-check/.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+
+language: python
+python: 
+  - "3.5"
+
+install:
+  - sudo apt-get install -y libssl-dev libcurl4-openssl-dev bsdmainutils vim net-tools inetutils-ping
+  - pip3 install pfurl
+
+script: 
+  - python3 automate.py
```

Comment:
I like the idea of validating changes to the script with travis, but you'll need to use config on travis to pass the creds